pkg/kv/kvserver/kvflowcontrol/rac2/range_controller.go

```diff
@@ -2514,6 +2537,10 @@ func (rs *replicaState) scheduledRaftMuLocked(
 		rss.tryHandleModeChangeRaftMuAndStreamLocked(ctx, mode, false, false)
 		return false, false
 	}
+	if rss.mu.sendQueue.forceFlushStopIndex.active() &&
+		rss.exceedInflightBytesThresholdRaftMuAndStreamLocked() {
```

Comment:
Have a helper for this in `rss`? There is another snippet like this around line 2610.